test/unit/menu-test.js

```diff
@@ -161,22 +157,19 @@ describe('Menu', () => {
             shakaIndex: 0,
             shakaId: 32
         };
-        settingsMenu = new SettingsMenu(api, viewModel, controlbar, localization);
         viewModel.set('audioTracks', [track]);
         expect(!!settingsMenu.children.audioTracks).to.be.false;
         viewModel.set('audioTracks', [track, track]);
         expect(!!settingsMenu.children.audioTracks).to.be.true;
     });
 
     it('Emits an event when a submenu is added', () => {
-        settingsMenu = new SettingsMenu(api, viewModel, controlbar, localization);
         settingsMenu.trigger = sinon.spy();
         viewModel.set('levels', [{ label: 'Auto' }, { label: '1080p' }]);
         expect(settingsMenu.trigger.calledWith('menuAppended')).to.equal(true);
     });
 
     it('Emits an event when a submenu is removed', () => {
-        settingsMenu = new SettingsMenu(api, viewModel, controlbar, localization);
```

Comment:
is there a danger in this test setup? reusing the same settings menu across all these tests could be problematic since it requires order to remain the same which might not always be the case. 

why did we remove the instantiations?

Reply:
It's already in the beforeEach block

Reply:
ahhh nice!